pkgs/desktops/mate/mate-power-manager/default.nix

```
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
      itstool
      libxml2
      libcanberra-gtk3
-     gnome3.gtk
+     gtk3
      gnome3.libgnome-keyring
```

```suggestion
     libgnome-keyring
```
In `all-packages.nix` `libgnome-keyring3` == `gnome3.libgnome-keyring`

@jtojnar Think we can phase out `libgnome_keyring` and `libgnome_keyring3` from `aliases.nix`
It's been long enough.

---

`libgnome-keyring` is still part of `gnome3` – I was too lazy to move it out since it is deprecated.

Dropping the old aliases is would be nice but I would not block this PR on it.

---

> `libgnome-keyring` is still part of `gnome3` – I was too lazy to move it out since it is deprecated.

Yeah, I wouldn't have either.

> Dropping the old aliases is would be nice but I would not block this PR on it.

Right, this should be fine as is.